lms/static/sass/shared/_fields.scss

```diff
@@ -12,7 +12,7 @@
         border-radius: 3px;
 
         span {
-            color: $gray-l1;
+            color: $gray-d1;
```

Comment:
Are you sure that this rule is scoped to just being used in Profile UI areas? The file name being ``lms/static/sass/shared/_fields.scss`` is making me a bit nervous.